### DISCUSSION TITLE
How to override the default z-index 0 on the canvas?

### DISCUSSION THREAD

**Discussion by alberto-s-ramos** [matteobruni/tsparticles#4060] [General]
Hey! This might be a dumb question, but is there anyway I can override the default "z-index: 0 !important" on the canvas?

**Comment by matteobruni** [comment]
You can read here more about `fullScreen` options: https://github.com/matteobruni/tsparticles/issues/1277#issuecomment-1008909977

**Reply by alberto-s-ramos** [reply]
Yep, managed to solve it. Tyvm! 🙌
